localenv/mock-account-servicing-entity/app/routes/mock-idp._index.tsx

```diff
@@ -1,5 +1,13 @@
 import ConsentScreen from '~/routes/consent-screen'
+import { json } from '@remix-run/node'
+import { useLoaderData } from '@remix-run/react'
+import { CONFIG } from '~/lib/parse_config.server'
+
+export function loader() {
+  return json({ idpSecret: CONFIG.idpSecret })
+}
```

Comment:
Just thinking out loud, should some of these requests (ie, those requested via ApiClient.*) be handled in the `action()` handler on the server side instead? Maybe I'm just not used to remix, but seems a bit odd to have to return a env variable (that's probably best to keep secret) to the front-end to be used for admin API requests.